qi-lib/on.rkt

```diff
@@ -4,7 +4,7 @@
          flow-lambda
          define-flow
          π
-         let/flow)
+         #;let/flow)
```

Comment:
Why sexp-comment and not simply delete? If we need them they will still exist in the git history later.